gen3-integration-tests/tests/test_graph_submit_and_query.py

```diff
@@ -1,20 +1,55 @@
+import json
 import os
 import pytest
+import requests
 
 from cdislogging import get_logger
 from gen3.auth import Gen3Auth
+from gen3.index import Gen3Index
 from gen3.submission import Gen3SubmissionQueryError
-
+from utils.gen3_admin_tasks import create_expired_token
+from services.indexd import Indexd
+from services.coremetadata import CoreMetaData
 from services.graph import GraphDataTools
-
+from pages.login import LoginPage
+from pages.core_metadata_page import CoreMetadataPage
 
 logger = get_logger(__name__, log_level=os.getenv("LOG_LEVEL", "info"))
 
 
+def validate_consent_codes():
+    """
+    Function to check if consent_codes is available in dictionary or not.
+    Used to skip test if consent_codes in not available.
```

Comment:
update the function name - this function doesn't validate anything. maybe something like `is_consent_codes_dictionary` or `should_run_consent_codes_test`

also update the docstring to describe the return value - in which case is it true, in which case is it false (if the function name is obvious enough like the examples above, this is not necessary)